content/academy/apify_platform/publishing-your-actor.md

```diff
@@ -0,0 +1,68 @@
+﻿---
+title: Publishing your actor
+description: Prepare your actor for Apify Store with a description and README file and learn how to make your actor available to the public.
+menuWeight: 1
+category: courses
+paths:
+  - publishing-your-actor
+---
+
+# [](#publishing-actor) Publishing your actor
```

Comment:
This file should really just be an introduction to the **Publishing your actor** section. Just a bit of an overview of what the section will cover. Maybe 50 lines absolute maximum imo